web/src/pages/share/[id].vue

```diff
@@ -76,9 +55,8 @@
       const data: ShareDataReturnResponse = await response.json()
       if (response.ok) {
         console.log('Data:', data)
-        loading.value = false
 
-        if (data.data.length === 0) {
+        if (!data.data) {
```

Comment:
Double checking, will this work on an empty array?

Reply:
No it won't, but `data` will be an object instead of an array from now on